From dee78f822d8a5b985e565e5c1ef42a59557aa8fc Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Fri, 7 Mar 2025 11:35:46 -0800
Subject: [PATCH] Remove whitespace in operator""

Clang 20+ errors about this whitespace.

Fixes
include/toml11/impl/../fwd/literal_fwd.hpp:22:26: error: identifier '_toml' preceded by whitespace in a literal operator declaration is deprecated [-Werror,-Wdeprecated-literal-operator]
|    22 | ::toml::value operator"" _toml(const char* str, std::size_t len);
|       |               ~~~~~~~~~~~^~~~~
|       |               operator""_toml

Upstream-Status: Submitted [https://github.com/ToruNiina/toml11/pull/285]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 include/toml11/fwd/literal_fwd.hpp   | 2 +-
 include/toml11/impl/literal_impl.hpp | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/toml11/fwd/literal_fwd.hpp b/include/toml11/fwd/literal_fwd.hpp
index e46612c..82f62b7 100644
--- a/include/toml11/fwd/literal_fwd.hpp
+++ b/include/toml11/fwd/literal_fwd.hpp
@@ -19,7 +19,7 @@ inline namespace literals
 inline namespace toml_literals
 {
 
-::toml::value operator"" _toml(const char* str, std::size_t len);
+::toml::value operator""_toml(const char* str, std::size_t len);
 
 #if defined(TOML11_HAS_CHAR8_T)
 // value of u8"" literal has been changed from char to char8_t and char8_t is
diff --git a/include/toml11/impl/literal_impl.hpp b/include/toml11/impl/literal_impl.hpp
index e8298c2..067d4ed 100644
--- a/include/toml11/impl/literal_impl.hpp
+++ b/include/toml11/impl/literal_impl.hpp
@@ -115,7 +115,7 @@ inline namespace toml_literals
 {
 
 TOML11_INLINE ::toml::value
-operator"" _toml(const char* str, std::size_t len)
+operator""_toml(const char* str, std::size_t len)
 {
     if(len == 0)
     {
